README.md

```diff
@@ -58,6 +60,35 @@ stateLayout.showState(contentState, transition)
 Check [example](https://github.com/freeletics/StateLayout/tree/master/example) app to see how it works.
 
 
+# Dependency
+Dependencies are hosted on Maven Central:
+
+```groovy
+implementation 'com.freeletics.statelayout:statelayout:1.0.0'
+```
+Keep in mind that this library is written in kotlin which means you also need to add `kotlin-stdlib` to a project using StateLayout.
```

Comment:
This isn't necessary. The Kotlin stdlib is a transitive dependency, there is no need for consumers to explicitly add it.